master/buildbot/test/unit/test_reporter_gerrit.py

```diff
@@ -70,6 +85,29 @@ def testSummaryCB(buildInfoList, results, status, arg):
                             (GERRIT_LABEL_VERIFIED, verified))
 
 
+@defer.inlineCallbacks
+def testSummaryCBDeferred(buildInfoList, results, master, arg):
+    success = False
+    failure = False
+
+    for buildInfo in buildInfoList:
+        if buildInfo['result'] == SUCCESS:
+            success = True
+        else:
+            failure = True
+
+    if failure:
+        verified = -1
+    elif success:
+        verified = 1
+    else:
+        verified = 0
+
+    result = yield makeReviewResult(str(buildInfoList),
+                                    (GERRIT_LABEL_VERIFIED, verified))
+    defer.returnValue(result)
```

Comment:
Ah I thoughts those callbacks where tests.

There is a convention that says that methods starting with test are tests.
Please rename those function, like testedSummaryCBDeferred